sdk/identity/Azure.Identity/src/CliCredentialClient.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Azure.Identity
+{
+    internal class CliCredentialClient
+    {
+        public virtual (string, int) CreateProcess(string extendCommand)
+        {
+            string fileName = string.Empty;
+            string argument = string.Empty;
+
+            Process proc = new Process();
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                fileName = "cmd";
+                argument = $"/c \"{extendCommand}\"";
```

Comment:
Do we need to escape shell-meta characters in any way from `extendedCommand` since it did not expect to invoked via `cmd`?

Reply:
This extendCommand is a combination of  "get account get-access-token --resource" and "Scopes". Can we double check with a person who knows "Scopes" clear?

Reply:
My concern is that someone could call `GetToken` directly, passing whatever scopes they want.  If that scope contains shell meta characters, it's possible that we could force `cmd` to run additional programs after the Azure CLI.  I don't know if this is something we need to be worried about.

Reply:
Looking at https://github.com/Azure/go-autorest/blob/b965d219bd9d10676384ef8f05da2e2441df9829/autorest/azure/cli/token.go#L125-L175 may be useful.  They do some validation of "resource" there (which for us would be the "scope" that's put into the extended command.  I think if we did that validation we'd be in good shape.